src/Symfony/Component/DependencyInjection/Tests/Fixtures/NamedArgumentsVariadicsDummy.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+namespace Symfony\Component\DependencyInjection\Tests\Fixtures;
+
+class NamedArgumentsVariadicsDummy
+{
+    public function __construct(...$variadics)
```

Comment:
for the tests, could you add another parameter before the variadic one so we show it works?